lisa/env.py

```diff
@@ -46,37 +47,46 @@
 FTRACE_BUFSIZE_DEFAULT = 10240
 RESULT_DIR = 'results'
 LATEST_LINK = 'results_latest'
-DEFAULT_DEVLIB_MODULES = ['sched', 'cpufreq', 'cpuidle']
 
 class TargetConf(MultiSrcConf, HideExekallID):
+    """
+    Target connection settings.
+
+    {generated_help}
+    """
     YAML_MAP_TOP_LEVEL_KEY = 'target-conf'
 
-    STRUCTURE = {
-        'kind': str,
-        'host': str,
-        'board': str,
-        'username': str,
-        'password': str,
-        'port': int,
-        'device': str,
-        'keyfile': str,
-        'workdir': str,
-        'tools': StrList,
-        'ftrace': {
-            'events': StrList,
-            'functions': StrList,
-            'buffsize': int,
-        },
-        'devlib': {
-            'platform': {
-                'class': str,
-                'args': Mapping,
-            },
-            'excluded-modules': StrList,
-        }
-    }
+    STRUCTURE = TopLevelKeyDesc(YAML_MAP_TOP_LEVEL_KEY, 'target connection settings', (
+        KeyDesc('board', 'Board name, free-form value only used to embelish logs', [str]),
```

Comment:
Since `board` has changed from affecting `TestEnv` to just being a pretty-print helper, it might be time to rename it to `name` or something similar.

Reply:
makes sense, that is what I actually already did for PlatformInfo